packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -589,19 +595,45 @@ export namespace EmrCreateCluster {
   }
 
   /**
-   * The launch specification for Spot instances in the fleet, which determines the defined duration and provisioning timeout behavior.
+   * The launch specification for On-demand instances in the instance fleet, which determines the allocation strategy.
+   *
+   * @see https://docs.aws.amazon.com/emr/latest/APIReference/API_OnDemandProvisioningSpecification.html
+   *
+   * @experimental
+   */
+  export interface OnDemandProvisioningSpecificationProperty {
```

Comment:
I have followed existing conventions of using Property throughout. I think changing this to Props would require changing existing names to Props as well and that would introduce a breaking change.

As for the experimental annotation, its used all over this file, is it fine to remove them all?

Reply:
sure, let's leave it alone for now. i'm not a fan of proliferating something we know we want to change because that's the way it's always been done.

but i'm happy to do it in another change. it will break more things, but that's the only way to stop the trend.